src/vm/dllimport.cpp

```diff
@@ -2950,6 +2960,236 @@ void PInvokeStaticSigInfo::DllImportInit(MethodDesc* pMD, LPCUTF8 *ppLibName, LP
     }
 }
 
+#if !defined(CROSSGEN_COMPILE) // IJW
```

Comment:
Do you want to use !CROSSGEN_COMPILE to enable IJW? do you plan to add a new feature switch for IJW?

Reply:
I wasn't planning any feature switch for IJW since there's no particular reason to disable it. However, some of its dependencies aren't available under crossgen, so parts of it have to be ifdefed for that.